spec/pb_kits/playbook/kits/title_count_spec.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require_relative "../../../../app/pb_kits/playbook/pb_title_count/title_count"
+
+RSpec.describe Playbook::PbTitleCount::TitleCount do
+  subject { Playbook::PbTitleCount::TitleCount }
+
+  it { is_expected.to define_partial }
+
+  describe "#classname" do
+    it "returns namespaced class name", :aggregate_failures do
+      expect(subject.new({}).classname).to eq "pb_title_kit_count_kit"
```

Comment:
looks like that first `kit` made it fail CI. Should just be  `"pb_title_count_kit"`, right?

Reply:
Looks like this PR is still labeled `wip`.

Reply:
Yeah, I'm still not working on it.